cobra/core/solution.py

```diff
@@ -0,0 +1,359 @@
+# -*- coding: utf-8 -*-
+
+"""Provide unified interfaces to optimization solutions."""
+
+from __future__ import absolute_import
+
+import time
+import logging
+
+from datetime import datetime
+from collections import OrderedDict
+from warnings import warn
+
+from builtins import (object, super, dict)
+from future.utils import python_2_unicode_compatible
+
+from cobra.exceptions import UndefinedSolution
+
+__all__ = ("Solution",)
+
+LOGGER = logging.getLogger(__name__)
+
+
+@python_2_unicode_compatible
```

Comment:
not used anywhere else, perhaps skip for now for consistency and then correct in the whole package if needed, reduce inconsistency?